app/assets/javascripts/components/stories/Stories.js

```diff
@@ -1,15 +1,15 @@
 import React, { Fragment } from "react";
 import StoryItem from "../story/StoryItem";
 
-const Stories = ({ stories }) => {
+const Stories = ({ stories, search = false }) => {
   if (!stories.length) {
     return null;
   }
 
   return (
     <Fragment>
       {stories.map(story => (
-        <StoryItem key={story.id} story={story} />
+        <StoryItem key={story.id} story={story} search={search} />
       ))}
     </Fragment>
   );
```

Comment:
E believe you don't need this `Fragment` here. You could rewrite this return:

```js
return stories.map(story => (
  <StoryItem key={story.id} story={story} search={search} />
))
```

Reply:
This way, you could remove:
```js
if (!stories.length) {
  return null;
}
```
if you set `stories` as an empty array by default:
```js
const Stories = ({ stories = [], search = false }) => {
```